src/dataLoaders/utils/pluginConfigs.ts

```diff
@@ -32,8 +33,19 @@ export const updateConfigFields = <T extends Plugin>(
   })
 }
 
-export const createNewPlugin = (name: TelegrafPluginName): Plugin => {
-  return telegrafPluginsInfo[name].defaults
+export const createNewPlugin = (plugin: TelegrafPlugin): Plugin => {
```

Comment:
i like this change. the method previously needed a name but had to be aware of the internal structure of the `telegrafPluginsInfo` data structure. this explicitness is better, imo because now it explicitly needs the `telegrafPluginsInfo` structure, rather than implicitly needing it